lib/learn_to_rank/reranker.rb

```diff
@@ -11,9 +11,8 @@ class Reranker
     include Errors
     # Reranker re-orders elasticsearch results using a pre-trained model
     def rerank(query: "", es_results: [])
-      feature_sets = FeatureSets.new.call(query, es_results)
-      new_scores   = Ranker.new(feature_sets).ranks
-
+      feature_sets = fetch_feature_sets(query, es_results)
```

Comment:
Typo this as `es_result` and all the tests return `55` rather than `nil`, that was strange

Reply:
🤔